bench-tps/src/main.rs

```diff
@@ -115,7 +115,7 @@ fn main() {
         // Sort keypairs so that do_bench_tps() uses the same subset of accounts for each run.
         // This prevents the amount of storage needed for bench-tps accounts from creeping up
         // across multiple runs.
-        keypairs.sort_by(|x, y| x.pubkey().to_string().cmp(&y.pubkey().to_string()));
+        keypairs.sort_by_key(|x| x.pubkey().to_string());
```

Comment:
@sakridge, @jstarry: I wonder why this is sorted in terms of base58...? Well, why `.to_string()` is needed? Plain binary byte-for-byte ordering isn't enough?

Reply:
@ryoqun I can't think of a good reason. binary comparison sounds good!

Reply:
I'll do this as a separate pr for back-porting.